zypp/target/TargetImpl.cc

```diff
@@ -1541,13 +1541,9 @@ namespace zypp
           {
             DBG << "dryRun/downloadOnly: Not installing/deleting anything." << endl;
             if ( explicitDryRun ) {
-              if ( policy_r.singleTransModeEnabled() ) {
-                // single trans mode does a test install via rpm
-                commitInSingleTransaction( policy_r, packageCache, result );
-              } else {
-                // if cache is preloaded, check for file conflicts
-                commitFindFileConflicts( policy_r, result );
-              }
+              // fall back to libsolv's fileconflict check because signeltrans
```

Comment:
```suggestion
              // fall back to libsolv's fileconflict check because singletrans would require packages to be locally available
```

Reply:
Yes, but I reverted the change. It's the `miss == false` case, so all packages are available.